safe/report/test/test_earthquake_report.py

```diff
@@ -0,0 +1,87 @@
+# coding=utf-8
+import os
+
+import io
+
+import shutil
+import unittest
+
+from jinja2.environment import Template
+
+from safe.common.utilities import safe_dir
+from safe.definitions.constants import ANALYSIS_SUCCESS
+from safe.definitions.reports.components import \
+    standard_impact_report_metadata_html
+from safe.impact_function.impact_function import ImpactFunction
+from safe.report.impact_report import ImpactReport
+from safe.report.report_metadata import ReportMetadata
+from safe.test.utilities import (
+    get_qgis_app,
+    load_test_vector_layer, load_test_raster_layer)
+
+QGIS_APP, CANVAS, IFACE, PARENT = get_qgis_app()
```

Comment:
I know it's new, but I think we should start using safe.test.utilities.qgis_iface() instead of get_qgis_app() It will make our life easier when will switch to QGIS3. For next time ;-)